README.md

```diff
@@ -1,7 +1,6 @@
 # ZIO keeper
 
 [![CircleCI][badge-ci]][link-ci]
-[![Gitter][badge-gitter]][link-gitter]
 
 ZIO keeper is a purely-functional, type-safe library for managing distributed state in a strong
```

Comment:
I would definitely change that to reflect reality. Something like "ZIO Keeper is purely-functional set of libraries for distributed programming in Scala." then we can add later that we have core that provides transport and membership in the cluster and we are working on Leader Election, CRDT, routing, partitioning and sharding.

Reply:
Draft?:)

Reply:
I know I know just don't want to forget about this :)